tests/test_basic.py

```diff
@@ -18,6 +18,7 @@
 from werkzeug.routing import RequestRedirect
 
 import flask
+TEST_PORT = 8080
```

Comment:
The idea of creating a constant for the test port value instead of changing it everywhere it exist is better from the code standard that we set ourselves. 🥇